app/views/venues/show.html.erb

```diff
@@ -4,4 +4,5 @@
   <%= link_to @venue.website, @venue.website, class: "website", target: "_blank" %>
   <p class="venue-address"><%= @venue.address %></p>
   <p class="venue-capacity">Capacity: <%= @venue.capacity %></p>
+  <p class="venue-t-accessible"><%= @venue.display_t_accessible %></p>
```

Comment:
Having logic in your view that determines if HTML is rendered or not is appropriate, that's what views are for. Having logic that calculates things (such as calculating the return value of the is_t_accessible  or display_t_accessible) is what you should (and did!) pull out into the model. I wouldn't recommend having a floating blank p class in your DOM, which is what a blank string would create